src/Pact/Gas/Table.hs

```diff
@@ -342,31 +342,31 @@ defaultGasModel = tableGasModel defaultGasConfig
 
 #if !defined(ghcjs_HOST_OS)
 -- | Costing function for binary integer ops
-intCost :: Integer -> Gas
-intCost !a
-  | (abs a) < threshold = 0
+intCost :: IntThreshold -> Integer -> Gas
+intCost ts !a
+  | abs a < threshold = 0
   | otherwise =
     let !nbytes = (I# (IntLog.integerLog2# (abs a)) + 1) `quot` 8
     in fromIntegral (nbytes * nbytes `quot` 100)
   where
   threshold :: Integer
-  threshold = (10 :: Integer) ^ (30 :: Integer)
```

Comment:
This constant should receive a clearer name, and perhaps be lifted out and up to the top of the file.

Reply:
`arbitraryConstant420`

I disagree with retroactively changing this. I'd rather simply use the `Threshold` type and annotate that.